ScoreSaber/Core/ReplaySystem/UI/Legacy/GameReplayUI.cs

```diff
@@ -89,103 +78,60 @@ internal class GameReplayUI : MonoBehaviour {
         }
 
         public string GetFriendlyModifiers(GameplayModifiers gameplayModifiers) {
-            switch (gameplayModifiers) {
```

Comment:
Good god, Re Sharper, what were you smoking when you did this?

Reply:
This could be moved to an helper class or something similar. LeaderboardUtils.cs also got the method GetModifierFromStrings. Both could probably be fused. I'm not sure if there's a better way.